File: Uri-Online-Judge/Wellington/C:C++/Iniciante/1048.cpp
#include <stdio.h>

int main(){
    double salario, salarioFinal, porcentual;
    
    scanf("%lf", &salario);
    
    if (salario <= 400) {
        porcentual = 0.15;
        salarioFinal = (salario * porcentual) + salario;
    } else if (400 < salario && salario <= 800) {
        porcentual = 0.12;
        salarioFinal = (salario * porcentual) + salario;
    } else if (800 < salario && salario <= 1200) {
        porcentual = 0.10;
        salarioFinal = (salario * porcentual) + salario;
    } else if (1200 < salario && salario <= 2000) {
        porcentual = 0.07;
        salarioFinal = (salario * porcentual) + salario;
    } else if (2000 < salario) {
        porcentual = 0.04;
        salarioFinal = (salario * porcentual) + salario;
    }
    
    char a[2] = {"%"};
    printf("Novo salario: %.2lf\n", salarioFinal);
    printf("Reajuste ganho: %.2lf\n", salarioFinal - salario);
    printf("Em percentual: %.0lf %s\n", porcentual * 100, a);
    
    return 0;
}

File: Uri-Online-Judge/Wellington/Python 3/Iniciante/1048.py
# -*- coding: utf-8 -*-

salario = float(input())

if (salario <= 400):
	percentual = 15
elif (salario <= 800):
	percentual = 12
elif (salario <= 1200):
	percentual = 10
elif (salario <= 2000):
	percentual = 7
else:
	percentual = 4

reajuste = salario * (percentual/100)
novoSalario = salario + reajuste

print('Novo salario: {:.2f}'.format(novoSalario))
print('Reajuste ganho: {:.2f}'.format(reajuste))
print('Em percentual: {} %'.format(percentual))